pipeline.py

```diff
@@ -99,8 +98,8 @@ def pipeline_wrapper(mock: List[Literal[MOCKED_STAGES]]):
 
     # Imports for evaluation
     from eval.final import run_final_eval_op
-    from eval.mmlu import load_mmlu_results_op, run_mmlu_op
 
+    # from eval.mmlu import load_mmlu_results_op, run_mmlu_op
     ## from eval.mmlu import run_mmlu_op, load_mmlu_results_op
```

Comment:
Why not just remove these 2 lines? Why comment them out?

Reply:
Removed, thanksç